Library/Homebrew/brew.sh

```diff
@@ -306,7 +313,17 @@ update-preinstall() {
 
   if [[ "$HOMEBREW_COMMAND" = "install" || "$HOMEBREW_COMMAND" = "upgrade" || "$HOMEBREW_COMMAND" = "tap" ]]
   then
+    if [[ -z "$HOMEBREW_VERBOSE" ]]
+    then
+      update-preinstall-timer & timer_pid=$!
+    fi
+
     brew update --preinstall
+
+    if [[ -n "$timer_pid" ]]
+    then
+      { kill "$timer_pid"; wait "$timer_pid"; } 2>/dev/null
```

Comment:
`SIGTERM` or `SIGKILL` doesn't make a difference to `sleep(1)`. I'm `wait`ing here only because it's good practice in general (at least to me), not because it's necessary.